Zend/zend_language_scanner.l

```diff
@@ -631,6 +634,27 @@ zend_op_array *compile_filename(int type, zval *filename)
 		convert_to_string(&tmp);
 		filename = &tmp;
 	}
+	if (!CG(script_extensions)[0]) {
```

Comment:
Since it checked before stream, it works as it supposed.
e.g. http://example.com/script.php (Allowed) http://exmple.com/script.myext (Not allowed)